client/src/components/MultiJurisdictionAudit/RoundManagement/index.tsx

```diff
@@ -95,16 +97,22 @@ const RoundManagement = ({ round, auditBoards, createAuditBoards }: IProps) => {
             >
               Download Ballot Labels for Round {roundNum}
             </FormButton>
-            {/* make conditional on online */}
-            <FormButton
-              verticalSpaced
-              onClick={() => downloadDataEntry(auditBoards)}
-            >
-              Download Audit Board Credentials for Data Entry
-            </FormButton>
-            <QRs passphrases={auditBoards.map(b => b.passphrase)} />
-            <RoundProgress auditBoards={auditBoards} round={round} />
-            {/* {online ? <RoundProgress /> : <RoundDataEntry />} */}
+            {online && (
```

Comment:
Also offline audits shouldn't have to create audit boards.

Reply:
@jonahkagan that's what I initially thought, but @mcchilders updated that part, unless I'm misinterpreting her.

Reply:
For now, just so I can test the behavior, I've sidestepped audit board creation.